### PR TITLE
[stable/sonatype-nexus] adding ACM cert annotations and LoadBalancer source IP whitelisting support

### PR DESCRIPTION
**Pull request by danielwhatmuff**
#### What this PR does / why we need it:

Adds ability to specify an AWS ACM certificate to provide SSL for the Nexus server.

#### Special notes for your reviewer:

Let me know if anything else is required/missing for this @rjkernick @tsiddique

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/sonatype-nexus/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: sonatype-nexus
-version: 1.19.0
+version: 1.19.1
```
Comment (rjkernick) [inline stable/sonatype-nexus/Chart.yaml]: Please bump the minor version as this adds features

File: stable/sonatype-nexus/values.yaml [hunk 2]
```diff
@@ -35,7 +35,16 @@ nexus:
   service:
     type: NodePort
     # clusterIP: None
-  # annotations: {}
+  annotations:
```
Comment (rjkernick) [inline stable/sonatype-nexus/values.yaml]: Don't we want the "{}" to remain the default?

**Review by rjkernick — CHANGES_REQUESTED** [review]

File: stable/sonatype-nexus/values.yaml (hunk 2 above)
Comment (danielwhatmuff) [reply stable/sonatype-nexus/values.yaml]: Yes, done

File: stable/sonatype-nexus/Chart.yaml (hunk 1 above)
Comment (danielwhatmuff) [reply stable/sonatype-nexus/Chart.yaml]: Done

**Comment by rjkernick** [comment]
Please fix the DCO: https://github.com/helm/charts/pull/16577/checks?check_run_id=212032508

**Comment by zanhsieh** [comment]
@rjkernick 
Could you approve this please? Thank you.

**Comment by rjkernick** [comment]
It doesn't look like either change was made from the review. Only the patch version is changed, and the {} are no longer the default value.

**Comment by danielwhatmuff** [comment]
Apologies. I will get on this soon.

**Comment by danielwhatmuff** [comment]
> It doesn't look like either change was made from the review. Only the patch version is changed, and the {} are no longer the default value.

Both done.

**Pull request merged**